src/main/java/Spaceship.java

```diff
@@ -0,0 +1,33 @@
+public class Spaceship implements Tradable, Drivable{
+    private int maxspeed;
+
+    public Spaceship() {
+        this.maxspeed = 200000000;
+    }
+
+    @Override
```

Comment:
Maybe add some documentation for the `upgradeSpeed` and `downgradeSpeed` methods. Why are we averaging the current speed and 300000000? Is it intentional that the speed will stop increasing once the current speed hits the constants?